### ISSUE TITLE
Scent Beast Concept Art

### ISSUE THREAD

**Issue by pyromaniac3** [GDACollab/Malisense#24]
### Important Notes: 
- Do NOT worry about creating finished and polished pieces! The iterative process is meant to have a quick turn around. Sketches and loose coloring is perfectly fine! I repeat _**DO NOT MAKE POLISHED PIECES!!**_ These are just concepts that only need to **convey the general idea**
- do not worry about fitting the style right now!! Concept art is all about the design of the monster. The style itself can be adapted to the design later on. 

## Your Task:
- Create monster concepts for the **Scent Beast**!!  For each design you make, jot a note about the specific thing you are experimenting with that drawing. Ex. What if they had a skull mask? 

## References
[More Pitch Art if you want to look at it](https://drive.google.com/drive/u/0/folders/1aWKugcB8eIZqMsT-yxiLpt1yt682Z95c)
[Visual Pitch](https://docs.google.com/presentation/d/19g8t3xpPiqEWSsNUM1sDcwQrgs8K4Ju9XZxHeyg2HuU/edit#slide=id.g25d196ab3fb_0_95)
[Written Pitch](https://docs.google.com/document/d/1W-QlBsmDG5n4nJ1vLYsneomtDKIt25F8z_fRuV0Bu2A/edit#heading=h.fsq3v84ywibm)

_**If you have any questions ask in the Malisense discord server's QNA and make sure to label your question under art!!**_

Good luck! and have fun <3

**Comment by tnguy510** [comment]
Taken by Trish Nguyen!

**Comment by HERECOMELUCNA** [comment]
This one fit here more then sight, so uh I think I change. laughing to heavy breathing 

![Image](https://github.com/GDACollab/Malisense/assets/156255711/33359ceb-1800-4795-9bdc-4d5f6f54e21a)

**Comment by HERECOMELUCNA** [comment]
updata

![Image](https://github.com/GDACollab/Malisense/assets/156255711/b901f67f-f8f7-4ad2-8052-c178e0e9d0b6)
![Image](https://github.com/GDACollab/Malisense/assets/156255711/08283e3b-9227-477b-9f15-461f99e95b49)

**Comment by HERECOMELUCNA** [comment]
![Image](https://github.com/GDACollab/Malisense/assets/156255711/b1085f04-7f93-4426-9917-d5c89f69fcbf)
![Image](https://github.com/GDACollab/Malisense/assets/156255711/10a073a6-9044-4475-a30d-c1c71b7cb1e3)
![Image](https://github.com/GDACollab/Malisense/assets/156255711/0327ba1f-12bf-4412-9333-e1c4874e0c3e)

**Comment by MikeMci00** [comment]
https://github.com/GDACollab/Malisense/issues/24#issuecomment-1899836167
- #2 being an Angerfish is super cool, would not have thought of that as the "scent," we could even go more in the direction of a Piranha (smelling blood in the water) by emphasizing its teeth. The way the tongue is represented is clever, too clever, as in the tongue is no longer the central focus and instead the light is what you are focusing on. Making the tongue thicker seems the obvious choice here.
- #3 is such a wretched creature (affectionate), its anatomical features suggest its a prey animal, while its presentation and distortion make it clear its a freaky predator. I have some more thoughts on predator animals I'll include in my Scent Beast art post tonight.
- (#1?) Tongue monster's most recent design is falling victim to too many complex details I think, because its very spaghetti like with a mix of different tentacle thicknesses its hard to keep track of and visually make sense of, and would be even harder to animation. Looking at your notes though I see what you mean about the tentacles being stripes like they are more tongue, I think that's a really cool direction that just needs more clarity. This could probably be resolved by only having two thin tentacles for grabbing, and removing or reworking the two chitinous hands (since they clash the most out of everything, unless you meant every hand is like that)

**Comment by tnguy510** [comment]
![Scent Beast 1st Concept](https://github.com/GDACollab/Malisense/assets/148594989/6f62ddae-643b-4bcb-98ab-b34d91897f3f)
Throwing my hat into the ring

**Comment by MikeMci00** [comment]
https://github.com/GDACollab/Malisense/issues/24#issuecomment-1901711439
- YES! Bug is a brilliant way to communicate to the player that the monster is using scent to detect them with it wriggling its antennae. Rather than relying on, say, just the classic "sniffing the air," this looks cooler and is terrifying when you see it happen!!
- Their design is too "cute" (circular and soft). Make them longer, ganglier, and more greebled (Inspiration: Rain World bugs)
- Antennae "emoting" is neat, should they be bigger?
- What if we made their tongue more apparent by making them a snake hybrid horror to contextualize why they also use a tongue when they are already a bug?

**Comment by WY-OwO** [comment]
Jocelyn Chiang (discord: wchiang3) joining in the task.

**Comment by MikeMci00** [comment]
Various takes on the Scent Beast, mainly with the theme of combing creatures that use scent.
Design 4 is based on @tnguy510 's "Bug + Snake" idea, and inclusion of more varied animals and marine life inspired by @HERECOMELUCNA 's ideas.
![Image](https://github.com/GDACollab/Malisense/assets/50094056/cf371a67-eced-45c2-b768-a2a1e46eefac)

**Comment by HERECOMELUCNA** [comment]
> [#24 (comment)](https://github.com/GDACollab/Malisense/issues/24#issuecomment-1899836167)
> 
> * Design Two being an Angerfish is super cool, would not have thought of that as the "scent," we could even go more in the direction of a Piranha (smelling blood in the water) by emphasizing its teeth. The way the tongue is represented is clever, too clever, as in the tongue is no longer the central focus and instead the light is what you are focusing on. Making the tongue thicker seems the obvious choice here.
> * Design Three is such a wretched creature (affectionate), its anatomical features suggest its a prey animal, while its presentation and distortion make it clear its a freaky predator. I have some more thoughts on predator animals I'll include in my Scent Beast art post tonight.
> * Design One. Tongue monster's most recent design is falling victim to too many complex details I think, because its very spaghetti like with a mix of different tentacle thicknesses its hard to keep track of and visually make sense of, and would be even harder to animation. Looking at your notes though I see what you mean about the tentacles being stripes like they are more tongue, I think that's a really cool direction that just needs more clarity. This could probably be resolved by only having two thin tentacles for grabbing, and removing or reworking the two chitinous hands (since they clash the most out of everything, unless you meant every hand is like that)

yep thank you for liking it, but the fish is underground and is kind of heat scent, I will now go fix the Tongue guy.

**Comment by MikeMci00** [comment]
https://github.com/GDACollab/Malisense/issues/24#issuecomment-1902527129
Ohh I see what you mean now about it being underground, thats actually an incredible de-scope idea then because it means the only time consuming part is it jumping out of the ground, unless it "surfed" up and down out of the ground partially. 
I think the lantern could benefit then from being more biological looking and giving the fish slightly longer crests to peak out of the ground like a fin.

**Comment by HERECOMELUCNA** [comment]
> [#24 (comment)](https://github.com/GDACollab/Malisense/issues/24#issuecomment-1902527129) Ohh I see what you mean now about it being underground, thats actually an incredible de-scope idea then because it means the only time consuming part is it jumping out of the ground, unless it "surfed" up and down out of the ground partially. I think the lantern could benefit then from being more biological looking and giving the fish slightly longer crests to peak out of the ground like a fin.

It good idea to see fin, I was going for a weird lamp is coming to you kind of ideas. Like unknown thing is coming your way and you don't know what heck it is, like you staying or running.

**Comment by HERECOMELUCNA** [comment]
I think all my creature it's just based on false safety illusion.

**Comment by niuxinss** [comment]
![image](https://github.com/GDACollab/Malisense/assets/156888628/b056b52c-fd73-4ae5-91cc-e652a2ca6342)

**Comment by MikeMci00** [comment]
https://github.com/GDACollab/Malisense/issues/24#issuecomment-1902782339
- You mentioned this to me, the chains to emphasize the slowness is really good. 
- I think it could use more end points for what the chains are actually connecting to instead of just being draped over, like a large collar and shackles on the limbs.
- I like having the eyes covered, but I think maybe the blanket makes the silhouette too friendly/not hostile enough, could maybe take inspiration from my Design One where the teeth and face are emphasized more?

**Comment by HERECOMELUCNA** [comment]
this better?

![Image](https://github.com/GDACollab/Malisense/assets/156255711/bb47dec3-0299-408c-ba7b-807f63198703)

**Comment by pyromaniac3** [comment]
https://github.com/GDACollab/Malisense/issues/24#issuecomment-1902782339 

hi! If you edit this post to include your First and last name, that would be great thank you!

**Comment by WY-OwO** [comment]
![Image](https://github.com/GDACollab/Malisense/assets/100245762/9dacd681-3616-4a4b-905c-0f94262ab4e2)

i just realized i forgot to put this general concept for scent beast here, i only remembered the google drive HHHH am I too late? Sorry...
